.github/workflows/build_and_test.yml

```diff
@@ -663,14 +661,14 @@ jobs:
         run: cmake --build "$BUILD_DIR" -j
 
       - name: Checkout ToyVpn-PcapPlusPlus
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # master
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           repository: seladb/ToyVpn-PcapPlusPlus
           path: ./ToyVpn-PcapPlusPlus
+          ref: "new-toyvpn"
```

Comment:
This will be removed once this PR is approved, and the new ToyVPN is released

Reply:
Updated in this commit: https://github.com/seladb/PcapPlusPlus/pull/1769/commits/b9e967269c5f2265d4ecaa224d67d85c48a5afc8